rtv/submission.py

```diff
@@ -244,33 +247,31 @@ def add_comment(self):
             curses.flash()
             return
 
-        # Fill the bottom half of the screen with the comment box
-        n_rows, n_cols = self.stdscr.getmaxyx()
-        box_height = n_rows // 2
-        attr = curses.A_BOLD | Color.CYAN
-
-        for x in range(n_cols):
-            y = box_height - 1
-            # http://bugs.python.org/issue21088
-            if (sys.version_info.major,
-                sys.version_info.minor,
-                sys.version_info.micro) == (3, 4, 0):
-                x, y = y, x
+        curses.endwin()
 
-            self.stdscr.addch(y, x, curses.ACS_HLINE, attr)
-
-        prompt = 'Enter comment: ESC to cancel, Ctrl+g to submit'
-        scol = max(0, (n_cols // 2) - (len(prompt) // 2))
-        self.stdscr.addnstr(box_height-1, scol, prompt, n_cols-scol, attr)
-        self.stdscr.refresh()
-
-        window = self.stdscr.derwin(n_rows-box_height, n_cols, box_height, 0)
-        window.attrset(Color.CYAN)
-
-        comment_text = text_input(window, allow_resize=False)
-        if comment_text is None:
+        fd, filename = tempfile.mkstemp(prefix="rtv-comment-", suffix=".txt")
```

Comment:
I think this can be simplified a lot using tempfile.NamedTemporaryFile

(untested)
```python
with NamedTemporaryFile(prefix="rtv-comment-", suffix=".txt") as fp:
    ...
    fp.write(comment_info)
    os.system('%s %s' % (os.getenv('EDITOR'), fp.name)
    comment_text = fp.read()
    ...
```
The file will automatically be closed at the end of the context manager and you can get rid of the try-except block.